### ISSUE TITLE
Live TV Guide screen - No Program Data Available

### ISSUE THREAD

**Issue by Migs3** [jellyfin/jellyfin-androidtv#3942]
### Describe the bug

Similar to #3791

From the main screen go down and select Live TV guide. 

Expect to see the guide data populate.

Instead all channels show 30 minute sections that say "No program data available", checked future date/time, and loading the next set of channels, no change. 

However... if you hit ok and go up and select the horizontal channel selector while already watching a channel, the guide data does actually shows correctly.

Also, on web and my Android phone app, they both show the correct guide info.

### Application version

0.17.4

### Where did you install the app from?

Amazon Appstore

### Device information

Fire TV Stick 4K

### Android version

Fire OS 6.7.0.5

### Jellyfin server version

10.9.9

**Comment by Migs3** [comment]
No change on jellyfin server 10.9.11.

**Comment by crobibero** [comment]
No, both my regular tv guide and tv guide while watching a channel work correctly.

**Comment by Hello-World-Traveler** [comment]
`FontLog : No resolver, short-circuit {Noto Color Emoji Compat, wdth 100.0, wght 400, ital 0.0, bestEffort false} for org.jellyfin.androidtv to failure [CONTEXT service_id=132 ]` 

Could this be why the app is not loading the TV Guide?

` procName:org.jellyfin.androidtv 2 callback:android.view.Choreographer$FrameDisplayEventReceiver,target:android.view.Choreographer$FrameHandler $ latencyTime:73,WallTime:413 android.os.MessageQueue.nativePollOnce:-2,android.os.MessageQueue.next:335,android.os.Looper.loop:200,android.app.ActivityThread.main:7848,java.lang.reflect.Method.invoke:-2,com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run:592,com.android.internal.os.ZygoteInit.main:1192, allocWait:-1,ionwait:-1,dstate:-1,sched:-1,iowait:-1 viewTitle:org.jellyfin.androidtv/org.jellyfin.androidtv.ui.browsing.MainActivity` delay in processing frame events?

**Comment by Hello-World-Traveler** [comment]
Installed Jellyfin 10.10.0, guide data available.

**Comment by Migs3** [comment]
I see no change with Server 10.10.1, and latest stable release 0.17.9

Guide data still all shows No Program Data Available when going into Guide.

**Comment by Hello-World-Traveler** [comment]
Must be with 0.18 beta
